src/native/crt.h

```diff
@@ -82,6 +82,13 @@ bool aws_jni_get_and_clear_exception(JNIEnv *env, jthrowable *out);
  ******************************************************************************/
 int aws_size_t_from_java(JNIEnv *env, size_t *out_size, jlong java_long, const char *errmsg_prefix);
 
+/*******************************************************************************
+ * Set a uint64_t based on a Long (jobject).
+ * If conversion fails, a java IllegalArgumentException is thrown like
+ * "{errmsg_prefix} cannot be negative" and AWS_OP_ERR is returned.
+ ******************************************************************************/
+int aws_uint64_t_from_java_Long(JNIEnv *env, uint64_t *out_long, jobject java_Long, const char *errmsg_prefix);
```

Comment:
ignore: blugh, I hate mixing error handling conventions in our CRT bindings (returning AWS_OP_ERR/SUCCESS, but it's a java exception that's been raised, not an aws_last_error())

but there's already lots of precedent for this, so it's fine but blugh

Reply:
Meme by chatgpt:
![image](https://github.com/awslabs/aws-crt-java/assets/7613045/da39a57c-9d19-40d5-9b46-6c55aa4e9cb4)